src/Utilities/tests/test_pooled_memory.cpp

```diff
@@ -91,6 +94,34 @@ TEST_CASE("pack scalar", "[utilities]")
   REQUIRE(i6[0]==i6[0]);
   REQUIRE(i6[1]==i6[1]);
   REQUIRE(i6[2]==i6[2]);
+
+  // up to 512 MB.
+  for(size_t size_scale=15; size_scale<30; size_scale++)
```

Comment:
This timing code should be moved to it's own unit test, and ifdef-ed out by default, since it doesn't check anything.